kolena/workflow/test_run.py

```diff
@@ -89,15 +89,15 @@ class TestRun(Frozen, WithTelemetry, metaclass=ABCMeta):
 
     model: Model
     test_suite: TestSuite
-    evaluator: Union[Evaluator, BasicEvaluatorFunction, None]
+    evaluator: Optional[Union[Evaluator, BasicEvaluatorFunction]]
```

Comment:
@munkyshi was there a reason this was previously a `Union[..., None]` instead of an `Optional[Union[...]]`? If so I will revert these drive-bys.